handlers/match.js

```diff
@@ -103,14 +103,26 @@ CSGO.CSGOClient.prototype.requestGame = function(matchid, outcome, token, callba
       payload.toBuffer(), callback);
 };
 
-CSGO.CSGOClient.prototype.requestRecentGames = function(accid, callback) {
-  callback = callback || null;
+CSGO.CSGOClient.prototype.requestRecentGames = function(arg1, arg2) {
   if (!this._gcReady) {
     if (this.debug) {
       util.log("GC not ready");
     }
     return null;
   }
+  
+  var accid = this.ToAccountID(this._user._client.steamID);
+  var callback;
+  if(arguments.length >= 2) {
```

Comment:
Just to keep everything nice and tidy, you'll probably want to follow the same if/else style and spacing that the rest of the code uses (i.e.

```
if (foo) {
    bar();
} else {
    foobar();
}
```

Besides that, looks good!